pkl-doc/src/main/kotlin/org/pkl/doc/DocPackageInfo.kt

```diff
@@ -159,7 +159,14 @@ data class DocPackageInfo(
   internal fun getModuleImportUri(moduleName: String): URI =
     when (importUri) {
       "pkl:/" -> "pkl:${moduleName.substring(4)}".toUri()
-      else -> "$importUri${getModulePath(moduleName, moduleNamePrefix)}.pkl".toUri()
+      else -> {
+        val path =
+          getModulePath(moduleName, moduleNamePrefix)
+            .split("/")
+            .map { it.uriEncoded }
+            .joinToString("/") { it } + ".pkl"
```

Comment:
`DocPackageInfo` only restricts `moduleNamePrefix` to `!contains("/")`. What are the odds someone will cheat a query component or fragment in there? Wondering also whether uses might spread and whether this logic of avoiding `/`s should live in what is now `uriEncoded` (although that might need renaming)

Reply:
It should be okay if they put other stuff in there; it will all get URI encoded in our generated file. A doc package whose name is `foo?bar=baz` is treated as path segment `foo%3Fbar%3Dbaz`